### DISCUSSION TITLE
how to set Errors for someone specific index on Array of errors

### DISCUSSION THREAD

**Discussion by julio3266** [react-hook-form/react-hook-form#7796] [Show and tell]
I have a form using useFieldArray and I need to insert error into an index inside the array. As you can see on the example below: 

The specific index is 1, in this case
<img width="392" alt="Captura de Tela 2022-02-10 às 22 00 34" src="https://user-images.githubusercontent.com/52897217/153522491-d00b9831-e3df-440b-bc28-a6a8be52ed2e.png">

**Comment by bluebill1049** [comment]
have you tried this API: https://react-hook-form.com/api/useform/seterror

**Reply by julio3266** [reply]
@bluebill1049 yes, but... I'm have a infinity loop but... I don't know if I'm doing making wrong.
my controller and input:
<img width="613" alt="Captura de Tela 2022-02-10 às 22 37 50" src="https://user-images.githubusercontent.com/52897217/153525384-001522f8-40b3-4abd-875b-3db2ce3d3ca9.png">
